src/test/java/utils/FileHelper.java

```diff
@@ -35,4 +43,12 @@ public static String readJsonFileAsString(String path) {
       throw new TestExecutionException(ioException.getMessage());
     }
   }
+
+  public static List<Product> readFileAsProduct(String path) {
+    try (Stream<String> br = new BufferedReader(new FileReader(path)).lines()) {
+      return br.skip(1).map(line -> new Product(line.split(PATH_SEPARATOR_REGEXP))).collect(Collectors.toList());
```

Comment:
As a recommendation, you can use the Jackson library to parse CSV files. For more details, please read this: https://cowtowncoder.medium.com/reading-csv-with-jackson-c4e74a15ddc1